coduno.go

```diff
@@ -13,73 +13,138 @@ import (
 
 	"github.com/coduno/app/controllers"
 	"github.com/coduno/app/mail"
+	"github.com/coduno/app/models"
 	"github.com/gorilla/mux"
+	"github.com/gorilla/sessions"
+	"github.com/m4rw3r/uuid"
 
 	"golang.org/x/net/context"
 
 	"google.golang.org/appengine"
+	"google.golang.org/appengine/datastore"
 	"google.golang.org/appengine/log"
 	"google.golang.org/appengine/urlfetch"
 )
 
 var gitlabToken = "YHQiqMx3qUfj8_FxpFe4"
+var sessionID = "SESSIONID"
+var store = sessions.NewCookieStore([]byte("c6fe0cba-5a74-478d-8389-a5d852c7fsae2b"))
```

Comment:
This should be a [secret](http://www.gorillatoolkit.org/pkg/sessions#NewCookieStore).